holidays/countries/india.py

```
@@ -603,3 +730,55 @@ class IndiaIslamicHolidays(_CustomIslamicHolidays):
+    name_guru_tegh_bahadur_martyrdom_day = tr("Guru Tegh Bahadur's Martyrdom Day")
+
+    # Magh Bihu.
+    name_magh_bihu = tr("Magh Bihu")
+
+    # Janmashtami (Smarta).
+    name_janmashtami_smarta = tr("Janmashtami (Smarta)")
+
+    special_public_holidays = {
+        # Dussehra (Mahanavami).
+        2002: (OCT, 14, tr("Dussehra (Mahanavami)")),
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)